ui/app/templates/components/job-editor/edit.hbs

```diff
@@ -6,7 +6,6 @@
 <div class="boxed-section">
   <div class="boxed-section-head">
     Job Definition
-    {{#if @data.cancelable}}
```

Comment:
The previous position of this `{{#if}}` meant that the word wrap would get hidden in cases of a brand new job template being written in the UI, when probably the only thing intended to be hidden was its neighbouring "cancel" button. Moved it as such.